Add command line options in About window - fixes #37

midwan opened the pull request:
- Extended the About window width to better fit the content
- Added the command line options in the end of the Description

CHerSun commented:
`-s` / `--seconds` option is wrong both on readme and in your PR. It's `-s:<seconds>` or `--seconds:<seconds>`, not `-s <seconds>`. See #80. Hopefully both locations would be fixed some day.

midwan commented:
Oh, thanks for that. I'll update my PR, though I don't think it will ever get merged.
This project seems abandoned, unfortunately.

But I can keep my fork updated and do any further fixes there, including upgrading it to the latest .NET version.

midwan commented:
Hm, after doing some testing, what you mention doesn't seem to be the case. At least not when starting this from the debugger.

If we use `-s:X` where `X` is a valid `int`, the parameter is not parsed and ends up as zero.
If we use `-s X` instead, we get the value.

midwan commented:
Reverted my changes in the PR

CHerSun commented:
Fought with this in the compiled binary, until I found #80. Code might have that fixed already, sorry, my bad.

CHerSun commented:
> But I can keep my fork updated and do any further fixes there, including upgrading it to the latest .NET version.

Would be cool, if you could upkeep the project with all those minor changes piling up. Especially if you can retarget scoop and chocolatey to your fork.

midwan commented:
I've updated mine to NET 9 already, fixed a few minor things here and there, and cherry-picked some of the suggested PRs here.

If there's enough interest, and if this repo is indeed abandoned (and not idle for some reason, which I fully respect), I could look into doing more, like chocolatey, winget etc.

vinaykapadia commented:
Please do! I'm also interested in having a fork of this project that is actively maintained.

cerebrate commented:
@midwan Actually, about that --

I've been meaning to get back to working on this project for literally years at this time, but other work in other areas keeps getting in the way - and, to be fair, it's no longer something I use for myself, so it tends to fall easily out of mine - which isn't exactly fair to the user community. I think everyone would agree that it could use a maintainer who has the time and energy to pay attention to it.

So if you'd like to be that maintainer, I'd be happy to give you the relevant access to this repository, or, heck, even just transfer the repo over to you lock, stock, and barrel.

Interested?

midwan commented:
@cerebrate 
Yes, I'm interested! :)

Let's discuss the details and find the best way forward. Can you drop me an e-mail? midwan at gmail dot com

Thank you!
